packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart

```diff
@@ -188,7 +187,7 @@ class MemoryEventsPaneState extends State<MemoryEventsPane>
 
   MemoryTimeline get _memoryTimeline => _memoryController.memoryTimeline;
 
-  ColorScheme colorScheme;
+  ColorScheme? colorScheme;
```

Comment:
I think you can remove colorScheme here too

Reply:
done